server/http_handler_test.go

```diff
@@ -155,6 +155,40 @@ func (ts *HTTPHandlerTestSuite) TestRegionIndexRange(c *C) {
 	}
 }
 
+func (ts *HTTPHandlerTestSuite) TestRegionCommonHandleRange(c *C) {
+	sTableID := int64(3)
+	indexValues := []types.Datum{
+		types.NewIntDatum(100),
+		types.NewBytesDatum([]byte("foobar")),
+		types.NewFloat64Datum(-100.25),
+	}
+	expectIndexValues := make([]string, 0, len(indexValues))
+	for _, v := range indexValues {
+		str, err := v.ToString()
+		if err != nil {
+			str = fmt.Sprintf("%d-%v", v.Kind(), v.GetValue())
+		}
+		expectIndexValues = append(expectIndexValues, str)
+	}
+	encodedValue, err := codec.EncodeKey(&stmtctx.StatementContext{TimeZone: time.Local}, nil, indexValues...)
+	c.Assert(err, IsNil)
+
+	startKey := tablecodec.EncodeCommonHandleSeekKey(sTableID, encodedValue)
```

Comment:
I found `EncodeCommonHandleSeekKey` is the same as `EncodeRowKey`, so I removed this function in #19320.

Reply:
OK. I see.